src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,17 @@
+:root {
+  font:
+    12px / 14px Roboto,
+    sans-serif;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0, 0, 0, 0) !important;
+  white-space: nowrap !important;
```

Comment:
But I don't understand why we need this selector. 
And why do we use 1px and -1px, not 0?

Reply:
Bootstrap has similar or equal class and rules like this one I used. I used this pack of rules from platform Mate Academy and I think it is a good practice to use it in this homework. You can ask my coach about that class

Reply:
Okay, I miss that. 
Thanks)